Documentation/Configuration/Index.rst

```diff
@@ -61,6 +61,8 @@ settings.latestComments                     Some settings for the latest comment
 settings.rss                                Basis rss-settings (see :code:`/Configuration/TypoScript/RSS/setup.txt` for page config)
 settings.backend                            Configure the backend output
 settings.debug                              Some debug configs
+settings.privacyPolicy.enabled              Enable the privacy policy checkbox in the subscription forms
```

Comment:
This says "forms" but it's actually only for the blog post subscription form, right?